app/javascript/components/Pages/CausesToCelebrate.js

```diff
@@ -7,10 +7,24 @@ import mentionsInputStyles from "../UI/mention/mentionsInputStyles";
 import BlockLowerBtns from "../UI/BlockLowerBtns";
 import CornerElements from "../UI/CornerElements";
 
-const CausesToCelebrate = ({data, setData, saveDataToDb, steps, service}) => {
+const CausesToCelebrate = ({data, setData, saveDataToDb, steps, service, draft}) => {
   const {response, users} = data
   const {isLoading, error} = service
   const [celebrateComment, setCelebrateComment] = useState(response.attributes.celebrate_comment || '')
+  const [isDraft, setDraft] = useState(draft);
```

Comment:
it is not clear with variables naming here as for me. If [0] variable is "isDraft" then [1] should be "setIsDraft".